proseco/acq.py

```diff
@@ -1282,6 +1286,22 @@ def get_spoiler_stars(stars, acq, box_size):
     return spoilers
 
 
+def force_even_and_on_ccd(val: int, offset: int) -> int:
+    """Force val to be even and on the CCD (0-1024).
+
+    Parameters
+    ----------
+    val : int
+        Input value.
+    offset : int
```

Comment:
Taking offset as an int and then using its sign feels a little weird, but I suppose since there's no native sign object to pass this makes sense and is better than a str.

Reply:
I thought about a bool (and maybe even tried it) but in the end this API looked more obvious in the calling code.